lib/soapy_cake/response.rb

```diff
@@ -22,6 +22,14 @@ def to_enum
       end
     end
 
+    def to_xml
```

Comment:
Are you sure you really want to use AdminBatched with the XML responses? I don't think that's a good idea. If we ever change its implementation (e.g. change batch size), we cannot use the archived responses anymore.